### PR TITLE
RTCMediaStreamTrackStats.concealedAudibleSamples added.

### LINKED ISSUE #202: RTCMediaStreamTrackStats.concealedAudibleSamples
In #187 we add RTCMediaStreamTrackStats audio samples. Should we have concealedAudibleSamples too?

### PR DESCRIPTION
**Pull request by henbos**
Fixes #202

### CODE REVIEW

**Comment by henbos** [comment]
Is it OK to have the user agent decide what is considered audible?

File: webrtc-stats.html [hunk 1]
```diff
@@ -1660,6 +1661,20 @@
...
+                <dfn><code>concealedAudibleSamples</code></dfn> of type <span class="idlMemberType">
+                <a>unsigned long long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only present for inbound audio tracks. The total number of concealed audio samples
+                  (see <a>concealedSamples</a>) that was played out during an audible portions of the
+                  stream. Audible means that the received audio is not considered background noise or
+                  silence by the user agent. It is up to the implementation to determine what is
+                  considered background noise.
```
Comment (vr000m) [inline webrtc-stats.html]: add: The value is set to undefined for user-agents that do not implement this metric.

Comment (henbos) [inline webrtc-stats.html]: Done!

**Comment by vr000m** [comment]
@jan-ivar should we wait for @jesup's feedback or are we good to go?

**Comment by vr000m** [comment]
Discuss this at TPAC, because we do not seem to move forward on this. Especially since the metric is not standardised anywhere . @dontcallmedom @aboba @stefhak

**Comment by vr000m** [comment]
@jesup @aboba bump! If you can have a look at this one before TPAC, it would definitely speed up discussion.

**Comment by vr000m** [comment]
TPAC decision was if there is a citable reference then lets move ahead, else this PR can be closed.

**Comment by henbos** [comment]
We don't have a citable reference. Sounds like a case for the incubator spec.

**Pull request closed without merging**